test/auth_oidc/test_auth_oidc.py

```diff
@@ -56,7 +58,7 @@ class OIDCTestBase(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls.uri_single = os.environ["MONGODB_URI_SINGLE"]
-        cls.uri_multiple = os.environ["MONGODB_URI_MULTI"]
+        cls.uri_multiple = os.environ.get("MONGODB_URI_MULTI")
```

Comment:
Should these all be `os.environ.get` for consistency?

Reply:
No, the others have to be there, we want it to error out.